src/video_core/renderer_opengl/renderer_opengl.h

```diff
@@ -59,6 +60,7 @@ class RendererOpenGL : public RendererBase {
     void DrawScreens();
     void DrawSingleScreenRotated(const ScreenInfo& screen_info, float x, float y, float w, float h);
     void UpdateFramerate();
+    void FrameLimiter(float average_frame_time, int frame_limit);
```

Comment:
then i guess i would have to target 59.9fps. then add a division in the function to get the correct diference